swagger/v3/api_spec.json

```diff
@@ -3021,15 +3794,219 @@
+          }
+        }
+      },
+      "ECFPartnershipUpdateDataRequest": {
+        "description": "Update An ECF partnership",
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "ecf-partnership-update"
+            ]
+          },
+          "attributes": {
+            "$ref": "#/components/schemas/ECFPartnershipUpdateAttibutesRequest"
+          }
+        }
+      },
+      "ECFPartnershipUpdateRequest": {
+        "description": "Update an ECF partnership",
+        "type": "object",
+        "required": [
+          "data"
+        ],
+        "properties": {
+          "data": {
+            "$ref": "#/components/schemas/ECFPartnershipUpdateDataRequest"
+          }
+        },
+        "example": {
+          "data": {
+            "type": "ecf-partnership-update",
+            "attributes": {
+              "delivery_partner_id": "db2fbf67-b7b7-454f-a1b7-0020411e2314"
+            }
+          }
+        }
+      },
+      "ECFSchool": {
+        "description": "An ECF school",
+        "type": "object",
+        "required": [
+          "id",
+          "type",
+          "attributes"
+        ],
+        "properties": {
+          "id": {
+            "description": "The unique identifier of the school",
+            "type": "string",
+            "format": "uuid",
+            "example": "cd3a12347-7308-4879-942a-c4a70ced400a"
+          },
+          "type": {
+            "description": "The data type",
+            "type": "string",
+            "example": "school"
+          },
+          "attributes": {
+            "$ref": "#/components/schemas/ECFSchoolAttributes"
+          }
+        }
+      },
+      "ECFSchoolAttributes": {
+        "description": "The data attributes associated with an ECF school",
+        "type": "object",
+        "required": [
+          "name",
+          "urn",
+          "cohort",
+          "in_partnership",
+          "induction_programme_choice",
+          "updated_at"
+        ],
+        "properties": {
+          "name": {
+            "description": "The name of the school",
+            "type": "string",
+            "example": "School Example"
+          },
+          "urn": {
+            "description": "The Unique Reference Number (URN) of the school",
+            "type": "string",
+            "example": "123456"
+          },
+          "cohort": {
+            "description": "The cohort the school is in",
+            "type": "string",
+            "example": 2021
+          },
+          "in_partnership": {
+            "description": "The cohort for which you are reporting the partnership",
+            "type": "boolean",
+            "example": true
+          },
+          "induction_programme_choice": {
+            "description": "The induction programme the school offers",
+            "type": "string",
+            "example": "not_yet_known",
+            "enum": [
```

Comment:
is "funded_induction_programme" not a value here? (Would cover schools doing FIP with lead providers)